### PR TITLE
project documentation

### PR DESCRIPTION
**Pull request by sadtoni**
Project documentation

### Author

Signed-off-by: Antonio Georgescu <mihai.georgescu2508@stud.fils.upb.ro>

### CODE REVIEW

**Comment by sadtoni** [comment]
@alexandruradovici

File: docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md
Comment (alexandruradovici) [inline docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md:1]: Please rename the folder with your username and the file *index.md*.

File: docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+|1 x [5 V Passive Buzzer](https://www.optimusdigital.ro/en/buzzers/634-5v-passive-buzzer.html?search_query=buzzer&results=87)|Honk |[2 RON](https://www.optimusdigital.ro/en/buzzers/634-5v-passive-buzzer.html?search_query=buzzer&results=87)|
+|1 x [10 mm Hexagonal Head Screw](https://www.optimusdigital.ro/en/screws-and-nuts/5756-m6x10-mm-hexagonal-head-screw.html?search_query=screw&results=746)|Fixing the components in place |[1 RON](https://www.optimusdigital.ro/en/screws-and-nuts/5756-m6x10-mm-hexagonal-head-screw.html?search_query=screw&results=746)|
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md:69]: Do you need these two libraries?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md (hunk 1 above)
Comment (sadtoni) [reply docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md:69]: no, i forgot to delete them from the template....... sorry🙃🙃🙃🙃🙃

**Comment by sadtoni** [comment]
@alexandruradovici cred ca e ok acum

File: docs/project/2024/mihai.georgescu2508/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Motivation
+
+The motivation behind my RC Car project comes from my desire to play, create and explore, and building the project gives me the opportunity to learn by building and coding everything from scratch.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/index.md:26]:
```suggestion
```

Add an image with your architecture here.

File: docs/project/2024/mihai.georgescu2508/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# RC Car controlled by a Web Application
+
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/index.md:2]:
```suggestion
# RC Car 
A RC Car controlled by a Web Application
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by sadtoni — COMMENTED** [review]
ok

**Comment by sadtoni** [comment]
image

File: docs/project/2024/mihai.georgescu2508/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+• On the bonnet, an ultrasonic sensor is placed to provide distance information to the user, allowing them to control the car even if it is out of sight.
+
+## Motivation
+
+The motivation behind my RC Car project comes from my desire to play, create and explore, and building the project gives me the opportunity to learn by building and coding everything from scratch.
+
+## Architecture 
+
+![image](https://github.com/sadtoni/upb-fils-ma.github.io/assets/122456673/a2cb09da-29d2-4d01-967c-65be150d6add)
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/index.md:24]: Please add the image to the repository.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mihai.georgescu2508/architecture schematic.jpg
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/architecture schematic.jpg:1]: Please do not use spaces in the file names.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mihai.georgescu2508/index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+:::
+
+## Description
+
+• Using the Raspberry Pi Pico W microcontroller, I built a remote-controlled car, operated via a web application.
+
+• The connection is realized with Pico’s integrated Wi-Fi capabilities, since it is ten times faster than using Bluetooth and is easier to control via a web browser on an iPhone.
+
+• On the bonnet, an ultrasonic sensor is placed to provide distance information to the user, allowing them to control the car even if it is out of sight.
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/index.md:16]: Please use markdown syntax for the lists.

File: docs/project/2024/mihai.georgescu2508/index.md (hunk 4 above)
Comment (alexandruradovici) [reply docs/project/2024/mihai.georgescu2508/index.md:24]: This is not solved, use the image from your repository with a relative path.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by sadtoni** [comment]
done

File: docs/project/2024/mihai.georgescu2508/index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+:::info 
+
+**Author**: Mihai-Antonio GEORGESCU \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-sadtoni/
+
+:::
+
+## Description
+
++Using the Raspberry Pi Pico W microcontroller, I built a remote-controlled car, operated via a web application.
```
Comment (alexandruradovici) [inline docs/project/2024/mihai.georgescu2508/index.md:12]: Did you test this using npm start?

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**